### DISCUSSION TITLE
Installing and cli command

### DISCUSSION THREAD

**Discussion by IOrlandoni** [dloebl/cgif#49] [Q&A] [answered]
Can I install this code as a package on Centos?   i am new on linux

**Comment by dloebl** [comment, marked as answer]
Hi @web-apply
cgif is relatively new, so it's not yet packaged for many distributions. But you may want to have a look here:
[https://repology.org/project/cgif/versions](https://repology.org/project/cgif/versions)

If cgif is not yet available for your Linux distribution, you can compile/install cgif as a shared library from source:

**Requirements:**
- C compiler (e.g. gcc, clang)
- [meson build system](https://mesonbuild.com/)

**Steps:**
For Centos the commands for building and installing cgif from source should look like this:

1. `sudo yum install meson gcc`
2. `git clone https://github.com/dloebl/cgif`
3. `cd cgif/`
4. `meson setup build --prefix=/usr`
5. `sudo meson install -C build`

I've also added general notes on the installation/compilation of cgif to [INSTALL.md](https://github.com/dloebl/cgif/blob/main/INSTALL.md). I hope this information is helpful.
